README.md

```diff
@@ -30,34 +29,43 @@ previous cluster managers such as [kops][kops] and
 - Support for single and multi-node control plane clusters
 - Deploy clusters with and without LBaaS available
 - Support for security groups
-- Doesn't use SSH for bootstrapping nodes
+- cloud-init based nodes bootstrapping
 
 ------
 
 ## Compatibility with Cluster API and Kubernetes Versions
 
 This provider's versions are compatible with the following versions of Cluster API:
 
-||Cluster API v1alpha1 (v0.1)|Cluster API v1alpha2 (v0.2)|
-|-|-|-|
-|OpenStack Provider v1alpha1 (release-0.1 branch)|✓||
-|OpenStack Provider v1alpha2 (v0.2)||✓|
+| abc  | def  | adfa  |adf   |asdf   |
+|---|---|---|---|---|
+| sdf  | sdfsdf  |sfddsf   |sdfsd   |sdf   |
+|   |   |   |   |   |
+|   |   |   |   |   |
+
+| | Cluster API v1alpha1 (v0.1) | Cluster API v1alpha2 (v0.2) | Cluster API v1alpha2 (v0.3) |
```

Comment:
should this be v1alpha3 ? (I don't pay attention to cluster-api release status ),so 0.3 is alpha2？

Reply:
fixed